widgets/bat.lua

```diff
@@ -1,10 +1,10 @@
 
 --[[
-												                        
-	 Licensed under GNU General Public License v2 
-	  * (c) 2013,      Luke Bonham                
-	  * (c) 2010-2012, Peter Hofmann              
-												                        
```

Comment:
@copycat-killer: Concerning writing style: this part was done automatically by my editor's config. Should I undo that? I tend to see spaces like this in other lua files.